server/index.js

```diff
@@ -36,11 +34,11 @@ app.get('/reviews', function (req, res) {
     // console.log('inside reviews-', locationId);
     // use the redis client 
     
-    // client.get(locationId, function (error, result) {
-    //     if (result) {
-            // const { getFive, totalReviews, searchResultsReviewsTotal, averageRatings } = result;
-        //     res.end(result);
-        // } else {
+    client.get(locationId, function (error, result) {
```

Comment:
shouldn't it be app.get, in order for the server to respond to get requests?

Reply:
Good question. This is Sujitha's code, I'll check in with her about this.